automation/vars/main.yml

```
@@ -366,12 +370,18 @@ pgbouncer_auth_user: true # or 'false' if you want to manage the list of users f
 pgbouncer_auth_username: pgbouncer # user who can query the database via the user_search function
 pgbouncer_auth_password: "" # If not defined, a password will be generated automatically during deployment
 pgbouncer_auth_dbname: "postgres"
-pgbouncer_client_tls_sslmode: "disable"
-pgbouncer_client_tls_key_file: ""
-pgbouncer_client_tls_cert_file: ""
+pgbouncer_client_tls_sslmode: "require"
+pgbouncer_client_tls_key_file: "{{ pgbouncer_conf_dir }}/server.key"
+pgbouncer_client_tls_cert_file: "{{ pgbouncer_conf_dir }}/server.crt"
 pgbouncer_client_tls_ca_file: ""
 pgbouncer_client_tls_protocols: "secure" # allowed values: tlsv1.0, tlsv1.1, tlsv1.2, tlsv1.3, all, secure (tlsv1.2,tlsv1.3)
-pgbouncer_client_tls_ciphers: "default" # allowed values: default, secure, fast, normal, all (not recommended)
+pgbouncer_client_tls_ciphers: "secure" # allowed values: default, secure, fast, normal, all (not recommended)
+pgbouncer_server_tls_sslmode: "require"
+pgbouncer_server_tls_protocols: "secure"
+pgbouncer_server_tls_ciphers: "secure"
+pgbouncer_server_tls_cert_file: "{{ pgbouncer_conf_dir }}/server.crt"
+pgbouncer_server_tls_key_file: "{{ pgbouncer_conf_dir }}/server.key"
```

As far as I know, we have to use the Postgres key and certificate here
```suggestion
pgbouncer_server_tls_cert_file: "{{ postgresql_conf_dir }}/server.crt"
pgbouncer_server_tls_key_file: "{{ postgresql_conf_dir }}/server.key"
```

server_tls_cert_file - Certificate for private key. PostgreSQL server can validate it.
server_tls_key_file - Private key for PgBouncer to authenticate against PostgreSQL server.

Source: https://www.pgbouncer.org/config.html

---

I might be wrong but I believe this is required if `clientcert=verify-ca` or `clientcert=verify-full` is set on `pg_hba.conf`. That way PostgreSQL can validate the cert and since it is the same it will accept it. Or, having different certs both signed by a CA (which we are not using). Will test your suggestion and get back.